design-system/uilib/src/components/dpdhl-toast/dpdhl-toast.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+    /**
+     * A Toast Component
+     * @component
+     */
+
+    /**
+     * Type of the toast: Error, Notification
+     */
+    import { Type } from './type'
+
+	export let open = true;
+    $: _open = open;
+
+    export let message;
+    $: _message = message;
+
+    export let type: string = Type.Error;
+    export let fullWidth: boolean = true;
+
+    export let title;
+    $: _title = title || "No title provided";
+
+    $: hasCustomIcon = Boolean($$slots["custom-icon"]) 
+
+    let toast: HTMLElement
+	function closeToast() {
+        toast.dispatchEvent(new Event('closeToast', {
+            bubbles: true,
+            composed: true,
+        }))
+	}
+
+
+</script>
+<svelte:options tag="dpdhl-toast" />
+
+{#if _open}
+    <main
+        class:notification = { type === Type.Notification }
+        class:error = { type === Type.Error }
+        class:full-width = {fullWidth}
+        bind:this={toast}
+        >
+        <div class="content-container">
+            <!-- TODO: hasCustomIcon is always false, why? -->
+            {#if hasCustomIcon} 
+                <div class="custom-icon">
+                    <slot class="hello" name="custom-icon"></slot>
+                </div>
+            {/if}
+            <div class="text-container">
+                <div class="title">
+                    {_title}
+                </div>
+                {#if _message}
+                    <div class="message">
+                        {_message}
+                    </div>
+                {/if}
+            </div> 
+        </div>
+
+        <dpdhl-icon on:click={closeToast} width={16} height={16} color='#FFF' icon="cancel" class="icon" />
+    </main>
+{/if}
+
+
+<style>
+    main{
+        display:        flex;
+        flex-direction: row;
+        align-items:    center;
+        padding:        var(--padding-top-bottom) var(--padding-left-right);
+        justify-content: space-between;
+    }
+
+    main.full-width{
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        flex-grow: 1;
+    }
+
+    main.notification {
+        color:         white;
+        background-color: var(--color-green-dark);
+    }
+
+    main.error {
+        color:         white;
```

Comment:
color is redundant and the same as `main.notification`, therefore could go into `main`
and with a css api:
```css
color: var(--toast__color, var(--color-white))
```